packages/lib/gpcircuits/circuits/proto-pod-gpc.circom

```diff
@@ -140,13 +151,44 @@ template ProtoPODGPC (
         isNullfierHashRevealed <== ownerIsNullfierHashRevealed
     );
 
+    /*
+     * 1 TupleModule with its inputs & outputs.
+     */
+
+    signal input tupleIndices[MAX_TUPLES][ARITY];
+    
+    signal tupleHashes[MAX_TUPLES] <== TupleModule(ARITY, MAX_ENTRIES, MAX_TUPLES)(entryValueHashes, tupleIndices);
+
+    /*
+     * 1 ListMembershipModule with its inputs & outputs.
+     */
+
+    // Index of entry value (if less than `MAX_ENTRIES`) or tuple hash that ought to be a member.
+    signal input memberIndex;
+
+    // Membership list.
+    signal input membershipList[MAX_LIST_ENTRIES];
+
+    var numEntriesAndTuples = MAX_ENTRIES + MAX_TUPLES;
+
+    if(MAX_LIST_ENTRIES > 0) {
```

Comment:
So the intent here is to use 0 to represent the absence of this module?
Makes sense, though if/when we scale to multiple lists that would be covered by 0 MAX_LISTS, at which point MAX_LIST_ENTRIES can be constrained to be >0.